hw16/index.ts

```diff
@@ -0,0 +1,20 @@
+import { customChecker } from "./custom-checker";
+
+ //positive checks
+ customChecker(2, 2, '+', 4);
+ customChecker(6, 3, '-', 3);
+ customChecker(10, 5, '/', 2);
+ customChecker(5, 5, '*', 25);
+ customChecker(100, 99, '-', 1);
+ customChecker(1, 1, '*', 1);
+ customChecker(90, 10, '/', 9);
```

Comment:
Please write your tests using test design technics. For example in the case of `+` operation you can check two positive numbers sum, negative and positive numbers sum, sum with `0`.